main.nf

```diff
@@ -1,234 +1,66 @@
-#!/usr/bin/env nextflow
-
-params.email = 'dec@u.northwestern.edu'
-params.directory = "$PWD/"
-params.out = params.directory.replace("raw", "processed")
-println params.out
-params.threads = 8
-println "Running Trimmomatic on " + params.directory
```

Comment:
These messages can be helpful when running

Reply:
`96d4a25`